manual/de/ch_Tools_Assistants.xml

```diff
@@ -4192,136 +4192,192 @@
+                  dass eine geplante Buchung im <xref linkend="sched-editor" /> angelegt ist, allerdings momentan
+                  keine Vorgänge zur Ausführung anstehen.
                 </para>
               </listitem>
             </varlistentry>
           </variablelist>
-        </listitem>
-      </varlistentry>
-    </variablelist>
+        </glossdef>
+      </glossentry>
+<!-- FIXME source: new item -->
+      <glossentry>
+        <glossterm>Wert</glossterm>
 
-    <para>At the bottom of the window there are two buttons:
-    </para>
+        <glossdef>
+          <para>Ist für die Eintragung in das Kontobuch noch die Angabe des Buchungswerts notwendig, so wird das mit
+            dem Hinweis <emphasis>(Wert benötigt)</emphasis> angezeigt. Dies kann z.B. bei einer
+            Buchung zwischen zwei Konten mit unterschiedlichen Währungen auf Grund von Kursschwankungen notwendig sein.
+          </para>
 
-    <itemizedlist>
-      <listitem>
-        <para>The <guibutton>Cancel</guibutton> button is used to exit the <guilabel>Since Last Run</guilabel>
-          window without creating the transactions scheduled to be entered.
-        </para>
-      </listitem>
+          <para>Sie können mit einem Doppelklick auf den Hinweistext ein Eingabefeld öffnen und einen Wert für
+            die Buchung eintragen. Das passiert ebenfalls, wenn Sie mit <guilabel>OK</guilabel> die
+            terminierte Buchung in ein Kontobuch übernehmen wollen.
+          </para>
+        </glossdef>
+      </glossentry>
 
-      <listitem>
-        <para>The <guibutton>Ok</guibutton> button closes the <guilabel>Since Last Run</guilabel> window and
-          applies the pending operations.
-        </para>
-      </listitem>
-    </itemizedlist>
+      <glossentry>
+        <glossterm>Erzeugte Buchungen durchsehen</glossterm>
 
-    <para>At the bottom right of the <guilabel>Since Last Run</guilabel> window there is also an option that
-      allows to <guilabel>Review created transactions</guilabel>. If this option is enabled when
-      there are one or more transactions <guilabel>To-create</guilabel>, pressing
-      <guibutton>OK</guibutton> will open a register window in which are shown the details of the
-      scheduled transactions splits entered automatically by &app;.
-    </para>
+        <glossdef>
+          <para>Wenn diese Option aktiviert ist, während eine oder mehrere Buchungen <guilabel>Zu
+            erstellen</guilabel> sind, wird durch Drücken von <guibutton>OK</guibutton> ein neues
+            Registerfenster geöffnet, in dem die Details der automatisch von &app; erstellten
+            Buchungsteile angezeigt werden und von Ihnen bearbeitet werden können.
+          </para>
+        </glossdef>
+      </glossentry>
+
+      <glossentry>
+        <glossterm>Hilfe</glossterm>
+
+        <glossdef>
+          <para>Öffnet diese Seite im Hilfe-Fenster.
+          </para>
+        </glossdef>
+      </glossentry>
```

Comment:
Ich glaube, diesen Glossentry brauchen wir öfters, z.B. ch_Transactions.xml:2992. Sollte man vielleicht zur Entität machen.

Ich denke in gnc-gui-*.dtd hinter `<!-- guilabel -->` als <!-- common glossentries based on gui-element -->

Wäre dann wohl eine separate PR.